applications/nrf_desktop/src/modules/hfclk_lock.c

```diff
@@ -27,7 +28,7 @@ static void hfclk_lock(void)
 		return;
 	}
 
-	clk_dev = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL);
+	clk_dev = device_get_binding(DT_INST_LABEL(0));
```

Comment:
? How can it point to CLOCK LABEL?
Why is `DT_DRV_COMPAT` added on top?

Reply:
https://docs.zephyrproject.org/latest/reference/devicetree/index.html#c.DT_INST_LABEL

equivalent would be `DT_LABEL(0, nordic_nrf_clock);`

Reply:
See also https://docs.zephyrproject.org/latest/guides/dts/api-usage.html#device-driver-conveniences in the [guide to the new devicetree API](https://docs.zephyrproject.org/latest/guides/dts/api-usage.html#device-driver-conveniences)